code/__DEFINES/maps.dm

```diff
@@ -158,7 +158,8 @@ Always compile, always use that verb, and always make sure that it works for wha
 #define CAMERA_LOCK_CENTCOM 4
 
 //Reserved/Transit turf type
-#define RESERVED_TURF_TYPE /turf/open/space/basic //What the turf is when not being used
+//VOID EDIT
+#define RESERVED_TURF_TYPE /turf/open/space //What the turf is when not being used
```

Comment:
Is there a reason we need this to not be basic?

Reply:
Basic space caused issues while loading, unloading planets. With normal space turfs it doesn't happen so this was added as a temporary solution to prevent the runtime spam.
Here are two atmos runtimes it causes
`Runtime in LINDA_turf_tile.dm, line 281: Cannot execute null.archive().
proc name: process cell (/turf/open/process_cell)`
`Runtime in LINDA_turf_tile.dm, line 346: Cannot execute null.reset cooldowns().`
I assume it's caused by ChangeTurf not being able to replace the turfs with basic space but I have no idea how it could be fixed.

Reply:
I've been testing with both and it seems the errors are the same, and unrelated, to basic space/space.